.gitignore

```diff
@@ -3,4 +3,4 @@ NEXT_RELEASE_CHANGELOG.md
 artifacts
 e2e/.env
 *.log
-.DS_STORE
\ No newline at end of file
+.DS_STORE
```

Comment:
This change is in two different PRs simultaneously, which could cause conflicts. Not a biggie most likely in this case, but it's usually easier to separate that out to its own PR if the same happens in the future.